### PR TITLE
MODKBEKBJ-244 API Tests : Provider | List of Packages | Display assigned package tags

### PR DESCRIPTION
**Pull request by stanislav-mureiko**
## Purpose

Adding api tests for the functionality implemented to display assigned package tags in a list of packages associated with a provider.
https://issues.folio.org/browse/MODKBEKBJ-244

## Approach

Adding new tests for particular endpoint.
Also I have deleted some extra tests request and transfer requests to right folder.

### CODE REVIEW

File: mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json [hunk 1]
```diff
@@ -3826,6 +3828,242 @@
...
+													"    pm.expect(response.data[0].id.split('-')[0]).eq(pm.environment.get(\"provider-with-tags-id\"));",
+													"});    ",
+													"",
+													"//Test that type is packages",
+													"pm.test('type is packages', function(){",
+													"    pm.expect(response.data[0].type).eq('packages');",
+													"});",
+													"",
+													"//Test that particular package has a correct tag list",
+													"pm.test('type is packages', function(){",
```
Comment (sduvvuri-ebsco) [inline mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json]: test should be renamed -> type is packages is name of the previous test

Comment (stanislav-mureiko) [reply mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json]: Thanks @sduvvuri-ebsco. I will fix it.

File: mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json [hunk 2]
```diff
@@ -4867,7 +5330,7 @@
 													"        let firstRecord = response.data[0];",
 													"        ",
 													"        //Test that type is providers",
-													"        pm.test('type is packages', function(){",
+													"        pm.test('type is provider', function(){",
```
Comment (sduvvuri-ebsco) [inline mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json]: This is a bit confusing -> are we checking for providers or packages here? comment says provider but we are checking that type should be packages?

Comment (stanislav-mureiko) [reply mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json]: I fixed it in previous commit. I changed this line to "provider".

File: mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json [hunk 3]
```diff
@@ -4953,7 +5416,7 @@
 													"        //Get the first record",
 													"        let firstRecord = response.data[0];",
 													"        let secondRecord = response.data[1];",
-													"        //Test that type is packages",
+													"        //Test that type is providers",
 													"        pm.test('type is providers', function() {",
```
Comment (sduvvuri-ebsco) [inline mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json]: same comment as above - are we checking for providers/packages?

Comment (stanislav-mureiko) [reply mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json]: Thanks. Already fixed.

File: mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json (hunk 2 above)
Comment (sduvvuri-ebsco) [reply mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json]: Still confused @stanislav-mureiko : on line 5333 -> pm.test('type is provider',) but on line 5334 -> pm.expect(firstRecord.type).eq('packages') we expect type to be packages? Sorry, I might be missing something here and thanks in advance for the clarification.

**Pull request merged**